IronMansChestLight/IronMansChestLight.ino

```diff
@@ -0,0 +1,38 @@
+/* Full documentation of library, @see https://learn.adafruit.com/adafruit-neopixel-uberguide/arduino-library */
+#include <Adafruit_NeoPixel.h>
+
+Adafruit_NeoPixel strip = Adafruit_NeoPixel(
+  12, /**< Number of LEDs                !!! <<< TODO: Adapt here! */
```

Comment:
@basejumpa I suggest to use a define or a const variable for both pin number and number of LEDs. Right now you have magic numbers.